pages/linux/bzip2.md

```diff
@@ -0,0 +1,15 @@
+# bzip2
+
+> A block-sorting file compressor, alias - bunzip2.
+
+- Compress file:
+
+`bzip2 {{path/to/file_to_compress}}`
+
+- Decompress file:
+
+`bzip2 -d {{path/to/compressed_file.bz2}}`
+
+- Decompress to STDOUT with 'bzip2 -dc' or use an alias 'bzcat':
```

Comment:
This is slightly cheating :) you could use the bzip2 command and just mention that bzcat is an alias in the description. Also, what does the "c" stand for?